std/variant.d

```diff
@@ -472,6 +472,14 @@ private:
                 auto rhsPA = getPtr(&temp.store);
                 return compare(rhsPA, zis, selector);
             }
+            // Generate the function bellow only if the Variant's type is
+            // comparable with 'null'
+            static if (__traits(compiles, () => A.init == null))
+            if (rhsType == typeid(null))
```

Comment:
The indenting is off here. I suggest putting the body of the `static if` inside braces.

Reply:
You still haven't indented this properly.

Reply:
oops, I forgot :sweat:

Reply:
done